extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
+    'triggers': {
+      'pageLoad': {
+        'on': 'visible',
+        'request': 'iframeMessage',
+        'iframePing': true,
```

Comment:
I'm also not a fan of `nativeConfig`, although it is fine with me if that is what you want :)

Would just `pageview` work?